src/careamics/careamist.py

```diff
@@ -52,19 +53,19 @@ class CAREamist:
 
     Attributes
     ----------
-    model : CAREamicsKiln
+    model : CAREamicsModule
         CAREamics model.
     cfg : Configuration
         CAREamics configuration.
     trainer : Trainer
         PyTorch Lightning trainer.
-    experiment_logger : Optional[Union[TensorBoardLogger, WandbLogger]]
+    experiment_logger : TensorBoardLogger or WandbLogger
```

Comment:
Should external types have the packages written explicitly? Or maybe for lightning types there can be a reference elsewhere in the docs.

Side note: I have realised this is not necessary for internal types because mkdocs will generate a link to the relevant docs!

Reply:
Yes mkdocstring is great!

Added the packages. It makes it a bit long but the pre-commit didn't complain about the two lines, so hopefully it will be fine once compile into the docs on the website!